packages/nuxt/src/core/plugins/import-protection.ts

```diff
@@ -19,7 +19,7 @@ export const vueAppPatterns = (nuxt: Nuxt) => [
   [/(^|node_modules\/)@vue\/composition-api/],
   ...nuxt.options.modules.filter(m => typeof m === 'string').map((m: any) =>
     [new RegExp(`^${escapeRE(m as string)}$`), 'Importing directly from module entry points is not allowed.']),
-  ...[/(^|node_modules\/)@nuxt\/kit/, /^nitropack/]
+  ...[/(^|node_modules\/)@nuxt\/kit/, /(^|node_modules\/)nuxt\/(config|kit|schema)/, /^nitropack/]
```

Comment:
Shall we include `nuxt` itself to the list or will it be conflicting with subpaths?

Reply:
it's already above on L17.